bower.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "url-join",
-  "version": "0.0.1",
+  "version": "1.0.0",
```

Comment:
(this is like a rule-of-thumb for node.js/ruby projects)

it is better to not bump the version of the module on the PR, because you don't know when is going to be merged.. I like to decide that after merging - before publishing. It is useful to put in the description of the PR something like "this breaks compatibility" etc,